### PR TITLE
Support mkdirp to create dest path if it doesn't exists	

### PR DESCRIPTION
**Pull request by watilde**
Fixes https://github.com/postcss/postcss-cli/issues/15

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -247,5 +249,11 @@ function writeFile(name, content, fn) {
     process.stdout.write(content);
     return fn();
   }
+
+  try {
+    fs.statSync(name);
+  } catch (e) {
+    mkdirp.sync(path.dirname(name));
+  }
```
Comment (TrySound) [inline index.js]: You can do better
```js
mkdirp(path.dirname(name), function (err) {
  if (err) {
    fn(err);
  } else {
    fs.writeFile(name, content, fn);
  }
});
```

Comment (TrySound) [inline index.js]: I mean even without precheck. with `fs.stat`

Comment (watilde) [inline index.js]: Seems nice! Let me update it to the above you wrote.

Comment (watilde) [inline index.js]: Thanks for your comment! Updated codes like you said @TrySound

**Comment by TrySound** [comment]
Can you provide test case?

**Comment by watilde** [comment]
Isn't this test I added enough to cover this patch? [Makefile#L95-L97](https://github.com/watilde/postcss-cli/blob/feature/mkdirp/Makefile#L95-L97)

**Comment by TrySound** [comment]
Oh, okay. Didn't understand it's test.

**Comment by watilde** [comment]
Yeah actually we can rewrite tests to js from `Makefile` to make it more readable :)

**Comment by watilde** [comment]
Merged this as https://github.com/postcss/postcss-cli/commit/f2fa45c32cf11ce88e43409bc547566e5cd3c85c and https://github.com/postcss/postcss-cli/commit/08f5539c3196630061f078d2040739060ab7e55e.

**Pull request closed without merging**